addons/patches/scifisupportplus/XEH_postInit_list.sqf

```diff
@@ -0,0 +1,50 @@
+#include "script_component.hpp"
+
+/* 
+ * Append SWS ground vehicles to the following variables:
+ * PHAN_UNSC_GV_list
+ * PHAN_UNSC_GV_listDISPLAY
+ *
+ * Follows pattern defined in PHAN_ScifiSupportPlus/bootstrap/xeh_postinit_list.sqf
+ */
+
+call compile preprocessFileLineNumbers 'PHAN_ScifiSupportPlus\bootstrap\xeh_postinit_list.sqf';
+
+private _sws_gv_list_insert = [
+	"SWS_V_APC_M107_Beaver",
+	"SWS_V_APC_M35_Cougar",
+	"SWS_V_MBT_M700_Viper"
+];
+
+{
+	if (_x!="" && (gettext (configFile >> "Cfgvehicles" >> _x >> "displayname")) != "") then 
```

Comment:
SQF does not do automatic boolean shortcircuiting unless something is included in a code block, e.g.
```sqf
if (false && throw "Error") then {} else {"OK"}; // throws an error, returns "OK"
if (false && { throw "Error"; }) then {} else {"OK"}; // does not throw an error, returns "OK"
```